components/annot-vocabulary.vue

```diff
@@ -323,20 +263,13 @@
 
             refreshTableItems() {
 
-                // This method generates the unique values that will be displayed in the UI for the user to annotate
-                // Two modes exist:
-                // row:     this mode shows the unique values in the relevantColumns (e.g. for diagnosis)
-                // column:  this mode shows the relevantColumn names and let's the user annotate each column
-                // In row mode, only values that are not declared as missing will be shown.
-
-                const tableArray = [];
-
-                // In 'row' mode create table entries for each value in the relevant columns
```

Comment:
I'm usually all for throwing way things but in this case I think keeping a high level comment about what the method does would be helpful.

Reply:
I'll place this back and adapt the comment for the current implementation.